src/spec/methods.cr

```diff
@@ -14,8 +14,8 @@ module Spec::Methods
   # ```
   #
   # If `focus` is `true`, only this `describe`, and others marked with `focus: true`, will run.
-  def describe(description, file = __FILE__, line = __LINE__, end_line = __END_LINE__, focus : Bool = false, &block)
-    Spec.root_context.describe(description.to_s, file, line, end_line, focus, &block)
+  def describe(description, file = __FILE__, line = __LINE__, end_line = __END_LINE__, focus : Bool = false, tags = nil, &block)
```

Comment:
Could you please add type restrictions for `tags` everywhere? It should be `String | Enumerable(String) | Nil`. This helps give better error messages if there is a mistake.

Reply:
Also I think all arguments after `description` should be named arguments. But that's a breaking change and should probably be a different PR.

Reply:
I don't understand the named args request.  I thought all params were by default callable with names in Crystal.

Reply:
You can force some to only be called by name. I think it's documented in the reference.

Reply:
Oh right...had to look it up as I've never used it, but it's this syntax:

```crystal
def foo(a, *, c)
```

There, after the splat, c is forced to be a named arg.

Reply:
Update with the type expansion.  I didn't do the named args though, as I think that should be a follow up PR since it should encompass more than just the tags parameter.